akka-docs/src/main/paradox/stream/stream-customize.md

```diff
@@ -301,8 +301,8 @@ In that sense, it serves a very similar purpose as `ActorLogging` does for Actor
 
 @@@ div { .group-java }
 
-You can extend the `akka.stream.stage.GraphStageWithLogging` or `akka.strea.stage.TimerGraphStageWithLogging` classes
```

Comment:
Good improvement. I think we can even do one better: by changing these to `@unidoc[GraphStageWithLogging]` they will be linked to the javadoc, and checked for typo's each time the docs are generated. Could you give that a try?

Reply:
Yep.
I added `@unidoc` for `GraphStageWithLogging` and `TimerGraphStageWithLogging `